src/rpc/net.cpp

```diff
@@ -530,18 +530,22 @@ UniValue setban(const JSONRPCRequest& request)
 
     if (strCommand == "add")
     {
-        if (isSubnet ? g_connman->IsBanned(subNet) : g_connman->IsBanned(netAddr))
+        if (isSubnet ? g_connman->IsBanned(subNet) : g_connman->IsBanned(netAddr)) {
```

Comment:
Unrelated. IMO remove change, otherwise fix others.

Reply:
I put it into a separate commit so `git blame` doesn't show an unrelated commit description in those lines.

I kept them however. Formatting changes adhere to the dev guide and so far we've been fixing up parts as we work touch during the work.